### PR TITLE
ARROW-7651: [CI][Crossbow] Nightly macOS wheel builds fail

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
@ursabot crossbow submit wheel-osx-cp37m

**Comment by kszucs** [comment]
[AMD64 Conda Crossbow Submit (#88213)](https://ci.ursalabs.org/#builders/98/builds/466) builder has been succeeded.

Revision: 38360212a30bb5ec75aac0532e90566572cf029a

Submitted crossbow builds: [ursa-labs/crossbow @ ursabot-456](https://github.com/ursa-labs/crossbow/branches/all?query=ursabot-456)

|Task|Status|
|----|------|
|wheel-osx-cp37m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|

**Comment by nealrichardson** [comment]
@ursabot crossbow submit wheel-osx-cp37m

**Comment by kszucs** [comment]
[AMD64 Conda Crossbow Submit (#88220)](https://ci.ursalabs.org/#builders/98/builds/467) builder has been succeeded.

Revision: 38360212a30bb5ec75aac0532e90566572cf029a

Submitted crossbow builds: [ursa-labs/crossbow @ ursabot-457](https://github.com/ursa-labs/crossbow/branches/all?query=ursabot-457)

|Task|Status|
|----|------|
|wheel-osx-cp37m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|

**Comment by kszucs** [comment]
Thanks @nealrichardson for restarting!

**Comment by kszucs** [comment]
@ursabot crossbow submit wheel-osx-cp37m

**Comment by kszucs** [comment]
[AMD64 Conda Crossbow Submit (#88221)](https://ci.ursalabs.org/#builders/98/builds/468) builder has been succeeded.

Revision: 754035f4d5689a498f90cf99317888cba121f728

Submitted crossbow builds: [ursa-labs/crossbow @ ursabot-458](https://github.com/ursa-labs/crossbow/branches/all?query=ursabot-458)

|Task|Status|
|----|------|
|wheel-osx-cp37m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|

**Comment by kszucs** [comment]
@ursabot crossbow submit wheel-osx-cp37m wheel-osx-cp38 wheel-osx-cp36m wheel-osx-cp35m wheel-osx-cp27m

**Comment by kszucs** [comment]
[AMD64 Conda Crossbow Submit (#88222)](https://ci.ursalabs.org/#builders/98/builds/469) builder has been succeeded.

Revision: 35069d187ac1e657b6f36f0715f2eedc0e17fe75

Submitted crossbow builds: [ursa-labs/crossbow @ ursabot-459](https://github.com/ursa-labs/crossbow/branches/all?query=ursabot-459)

|Task|Status|
|----|------|
|wheel-osx-cp27m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|
|wheel-osx-cp35m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|
|wheel-osx-cp36m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|
|wheel-osx-cp37m|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|
|wheel-osx-cp38|[TravisCI](https://travis-ci.org/ursa-labs/crossbow/branches)|

File: dev/tasks/python-wheels/osx-build.sh [hunk 1]
```diff
@@ -194,9 +194,8 @@ function install_wheel {
     pushd $1  # enter arrow's directory
     wheelhouse="$PWD/python/dist"
 
-    # Install compatible wheel
-    pip install $(pip_opts) \
-        $(python $multibuild_dir/supported_wheels.py $wheelhouse/*.whl)
+    # Install wheel
+    pip install $(pip_opts) $wheelhouse/*.whl
```
Comment (kszucs) [inline dev/tasks/python-wheels/osx-build.sh]: Something must have changed with the `wheel` package version, [supported_wheels.py](https://github.com/matthew-brett/multibuild/blob/devel/supported_wheels.py) was also updated lately.

We build single wheel per travis job, so filtering is not required and `pip install` should fail if the wheel is not suitable for the platform.

**Review by kou — APPROVED** [review]
+1

**Pull request closed without merging**